### ISSUE TITLE
How to improve accuracy

### ISSUE THREAD

**Issue by anguoyang** [davidsandberg/facenet#27]
Hi, David,
Just as the faceNet paper described, the accuracy could be reached over 99%, my question is, how to train the data to get this accuracy, is there any pre-trained model?

**Comment by davidsandberg** [comment]
Hi,
Yes, I guess this is the main question and I don't have the final answer. But I think part of the answer will be to use a larger set of training data, for example a combination of the VGG Face dataset and FaceScrub. But it will most likely include a larger network (including larger input images). 
In addition to this there will most certainly be other things that help to improve performance, like using dropout, use augumented training data (flipping and translations), maybe better weight initialization etc etc.

**Comment by anguoyang** [comment]
Hi, David,
Thanks for the answer.

is there pre-trained model used in the paper? as the paper claimed 99％, they must have done some test on their model.

**Comment by davidsandberg** [comment]
I'm sure they did a lot of experiments to evaluate different alternatives but I don't think there is a pre-trained model available from the authors.
